src/util.ts

```diff
@@ -93,7 +93,11 @@ export function parseContributorAccount(bodys: (string | null)[]): ContributorAc
   }
 
   return {
-    error: `Contributor did not properly post their account address.\n\nMake sure the pull request description (or user bio) has: "{network} address: {address}".`,
+    error: `hey 👋, you were nominated to receive a tip for your contribution 🤩
```

Comment:
This claims seems to be wrong to me. Just because we think that you should get a tip, doesn't mean that they will get a tip. A tip needs to be approved by OpenGov. At the same time they could learn that they can just create their own tip proposals.

Reply:
Change it now to mention OpenGov approval.